tests/slepc/solve_01.cc

```diff
@@ -62,8 +62,10 @@ check_solve( SolverType &solver,
       deallog << "Exception: " << e.get_exc_name() << std::endl;
     }
 
-  deallog << "Solver stopped after " << solver_control.last_step()
-          << " iterations" << std::endl;
+  // TODO make this robust on different platforms. Seems related to GHEP
+  // as solve_04 works ok.
+  //deallog << "Solver stopped after " << solver_control.last_step()
+  //        << " iterations" << std::endl;
```

Comment:
was not aware of that macro. 

I think the ranges are a bit unpredictable even though i start from the same random vectors and try to specify as much parameters as possible. I have a feeling there is still something left that SLEPc decides on its own and thus the discrepancy.